packages/mdc-button/_mixins.scss

```diff
@@ -227,27 +228,48 @@ $mdc-button-ripple-target: ".mdc-button__ripple";
   }
 }
 
-@mixin mdc-button-shape-radius($radius, $rtl-reflexive: false, $query: mdc-feature-all()) {
-  @include mdc-shape-radius(
-    mdc-shape-resolve-percentage-radius($mdc-button-height, $radius),
-    $rtl-reflexive,
-    $query: $query
+///
+/// Sets density scale for button.
+///
+/// @param {Number | String} $density-scale - Density scale value for component. Supported density scale values `-3`,
+///     `-2`, `-1`, `0`.
+/// @param {Boolean} $shape-rtl-reflexive - Set this to true to reflect the shape border radius in RTL context.
+///
+@mixin mdc-button-density($density-scale, $query: mdc-feature-all()) {
+
+  $height: mdc-density-prop-value(
+    $density-config: $mdc-button-density-config,
+    $density-scale: $density-scale,
+    $property-name: height,
   );
 
-  #{$mdc-button-ripple-target} {
-    @include mdc-shape-radius(
-      mdc-shape-resolve-percentage-radius($mdc-button-height, $radius),
-      $rtl-reflexive,
-      $query: $query
-    );
+  @include mdc-button-height($height, $query: $query);
+}
+
+@mixin mdc-button-height($height, $query: $query) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-feature-targets($feat-structure) {
+    height: $height;
   }
+}
+
+@mixin mdc-button-shape-radius($radius,
+$rtl-reflexive: false,
+$density-scale: $mdc-button-density-default-scale,
+$query: mdc-feature-all()) {
```

Comment:
nit: Indent +2

Reply:
![image](https://user-images.githubusercontent.com/47312/64626930-b970ca80-d3bc-11e9-9597-9e5a2e6daabd.png)

Stylelint throws error for +2 indentation on params.